### DISCUSSION TITLE
Odd erronous result

### DISCUSSION THREAD

**Discussion by chengtie** [jsvine/pdfplumber#1045] [Ask for help with specific PDFs]
Hello,

I have been using the following code to parse PDFs.

Given `page` as a parameter,
```
            explicit_horizontal_lines = set().union(*((row.bbox[1], row.bbox[3]) for row in (page.find_tables())[0].rows))
            explicit_horizontal_lines.add(max(page.vertical_edges, key=operator.itemgetter("bottom"))["bottom"])

            table = page.extract_table(dict(
                explicit_horizontal_lines=explicit_horizontal_lines, 
                horizontal_strategy="explicit"
            ))

            print(table)
```

For most of the PDFs, it returns good results to work on. For instance, for this [good.pdf](https://github.com/jsvine/pdfplumber/files/13453009/good.pdf), it returns the following `table`:
`[['Recorders Index Inquiry\nDisclaimer: Documents recorded as of close of business 11/11/2023. However indexing may not be current through this date\n224 Pages with 5,589 Documents\nDates: 04/29/2022 - 11/12/2023\n1 2 3 4 5 6 7 8 9 … 224 next ›', None, None, None, None, None, None, None], ['', 'Grantor:1/Grantee:2\nType/Name', 'Document Type\nLegal Description', 'Book\nPage', 'Real-Estate', 'Recorded Date\nInstrument Date', 'Document\nRelated', 'PDF'], ['', '1 Peterson Robert W Est\n1 Willemsen Janet M Ex\n2 Ipe1031 Rev513 Llc', 'Groundwater Hazard Stmt\n9-77-27 NE', '2023\n2760', '', '11/09/2023\n11/08/2023', 'GW2760\nRelated', ''], ['', '1 Benge Norma Peterson\n1 Peterson Norma\n1 Peterson Norma Benge\n2 Faux Lester Floyd\n2 Faux Peral Lynn', 'Groundwater Hazard Stmt\nPATTERSON ORIGINAL TOWN LOT 3 BLOCK 7\nPATTERSON ORIGINAL TOWN LOT 4 BLOCK 7', '2023\n2751', '', '11/09/2023\n10/30/2023', 'GW2751\nRelated', ''], ['', '1 Sandegren Jason\n1 Sandegren Jason R\n2 Douch Darey S\n2 Souksaengmany Chansamo', 'Affidavits', '2023\n2769', '', '11/09/2023\n10/31/2023', '2769', ''], ['', '1 Union State Bank\n2 Armstrong Linda S\n2 Armstrong Rick', 'Release Mortgage\n16-75-27 TRACT IN', '2023\n2768', '', '11/09/2023\n11/09/2023', '2768\nRelated', ''], ['', '1 Union State Bank\n2 Armstrong Linda S\n2 Armstrong Rick', 'Release Mortgage\n16-75-27 TRACT IN', '2023\n2767', '', '11/09/2023\n11/09/2023', '2767\nRelated', ''], ['', '1 Rosencrants Elaine L Tr\n1 Rosencrants Elaine Trust\n2 Rosencrants Elaine L Tr\n2 Rosencrants Elaine L Trust', 'Quit Claim Deed\n2-76-28 NE\n2-76-28 E2 W2 SE TRACT IN', '2023\n2766', '', '11/09/2023\n11/09/2023', '2766', ''], ['', '1 Lincoln Savings Bank\n2 Krish Investment Llc', 'Warranty Deed\nWINTERSET JEFFERSON PLACE LOT 101\nWINTERSET JEFFERSON PLACE LOT 102\nWINTERSET JEFFERSON PLACE LOT 103\nWINTERSET JEFFERSON PLACE LOT 104\nWINTERSET JEFFERSON PLACE LOT 106\nWINTERSET JEFFERSON PLACE LOT 107\nWINTERSET JEFFERSON PLACE LOT 108\nWINTERSET JEFFERSON PLACE LOT 201\nWINTERSET JEFFERSON PLACE LOT 202\nWINTERSET JEFFERSON PLACE LOT 203\nWINTERSET JEFFERSON PLACE LOT 204\nWINTERSET JEFFERSON PLACE LOT 205\nWINTERSET JEFFERSON PLACE LOT 206\nWINTERSET JEFFERSON PLACE LOT 207\nWINTERSET JEFFERSON PLACE LOT 208\nWINTERSET JEFFERSON PLACE LOT 302\nWINTERSET JEFFERSON PLACE LOT 303\nWINTERSET JEFFERSON PLACE LOT 304\nWINTERSET JEFFERSON PLACE LOT 305\nWINTERSET JEFFERSON PLACE LOT 306\nWINTERSET JEFFERSON PLACE LOT 307\nWINTERSET JEFFERSON PLACE LOT 308\nWINTERSET ORIGINAL TOWN LOT 6 BLOCK 9\nWINTERSET ORIGINAL TOWN LOT 7 BLOCK 9\nWINTERSET ORIGINAL TOWN LOT 8 BLOCK 9', '2023\n2765', '', '11/09/2023\n11/01/2023', '2765\nRelated', ''], ['', '1 Stine Seed Farm Inc\n2 Thompson Carrie\n2 Thompson Kyle', 'Warranty Deed\n13-74-29 E4 NE NW NE PARCEL C\n13-74-29 NW NE NE PARCEL C', '2023\n2764', '', '11/09/2023\n11/08/2023', '2764\nRelated', ''], ['', '1 Ipe 1031 Accommodators Llc\n1 Ipe1031 Rev513 Llc\n1 Lauterbach Carrine Lynn', 'Mortgage\n9-77-27 NE', '2023\n2763', '', '11/09/2023\n11/09/2023', '2763', '']]`

However, for this PDF [problem.pdf](https://github.com/jsvine/pdfplumber/files/13453001/problem.pdf), it returns a problematic result, which is hard to work on:

`[['1', '2', '3', '4', '5', '6', '7', '8', 'next ›', ''], ["Recorded Date\nGrantor:1/Grantee:2 Document Type Book Real- Document\nInstrument PDF\nType/Name Legal Description Page Estate Related\nDate\n1 First Iowa State Bank Release Mortgage 2023 11/22/2023 1263\n2 Mcgee James Edward 1263 11/21/2023 Related\n2 Mcgee Patricia Claire\n1 Stewart James L Warranty Deed 2023 11/22/2023 1262\n2 Stewart James L Trustee 72-16-08 NW PT 1262 11/21/2023\n2 Stewart James Lee Revocable Trust 72-16-08 NE SW\nAgreement\n1 Sinclair Linda L Affidavit/with Transfer 2023 11/22/2023 1261\n2 Sinclair Linda L ALB MOCK'S 1ST ADD BLK 2 LOT 1 1261 11/20/2023 Related\n2 Sinclair Mickey T ALB MOCK'S 1ST ADD BLK 2 LOT 2\n1 Vanzante Greg Alan Quit Claim Deed 2023 11/22/2023 1260\n1 Vanzante Jon Anthony 71-18-36 SW SE 1260 11/17/2023\n2 Vanz Llc 71-18-36 E2 SE SW PT\n71-18-36 SE SE\n1 Beary Shelly Revocable Trust Warranty Deed 2023 11/20/2023 1259\n1 Conner Shelly Renne Trustee 73-17-04 SW SE EX PARCEL B 1259 11/20/2023\n2 Conner Shelly Renne Trustee 73-17-04 NW SE\n2 Conner Shelly Revocable Trust 73-17-04 SE SE LOT 2 EX PARCEL B\n73-18-17 NW PT\nLOV MERRILL'S RAILROAD ADD BLK 7 LOT 2 & ALLEY\nADJ\nLOV MERRILL'S RAILROAD ADD BLK 6 LOT 3 & ALLEY\nADJ\n1 Beary Shelly Revocable Trust Affidavits 2023 11/20/2023 1258\n1 Conner Shelly Renne Trustee 73-17-04 SW SE EX PARCEL B 1258 11/20/2023 Related\n2 Beary Shelly 73-17-04 NW SE\n2 Beary Shelly Revocable Trust 73-17-04 SE SE LOT 2 EX PARCEL B\n2 City Of Lovilia 73-18-17 NW PT\n2 Conner Shelly Renne LOV MERRILL'S RAILROAD ADD BLK 7 LOT 2 & ALLEY\nADJ\nLOV MERRILL'S RAILROAD ADD BLK 6 LOT 3 & ALLEY\nADJ\n1 New Residential Mortgage Llc Release Mortgage 2023 11/20/2023 1257\n2 Anderson Jacquelyn 1257 11/20/2023 Related\n2 Anderson Larry Dean\n1 Albertson Jason A Mortgage 2023 11/20/2023 1256\n1 Albertson Trisha 72-16-20 SE SE PT 1256 11/17/2023\n2 First Iowa State Bank 72-16-29 NE NE PT\n1 Sample Carol Warranty Deed 2023 11/20/2023 1255\n1 Sample Christopher Lee 72-16-20 SE SE PT 1255 11/15/2023\n2 Albertson Jason 72-16-29 NE NE PT\n2 Albertson Trisha Marie\n1 Mcdanel Jean Administrator Court Officer's Deed 2023 11/20/2023 1254\n1 Mcdanel Stuart S Estate 71-17-35 W2 SW PT 1254 11/15/2023\n2 Mcdanel Shawn\n1 Mcdanel Jean Administrator Court Officer's Deed 2023 11/20/2023 1253\n1 Mcdanel Stuart S Estate 71-17-35 W2 SW PT 1253 11/15/2023\n2 Mcdanel Chad\n1 Mcdanel Jean Administrator Court Officer's Deed 2023 11/20/2023 1252\n1 Mcdanel Stuart S Estate 71-17-35 W2 SW PT 1252 11/15/2023\n2 Mcdanel Jean", None, None, None, None, None, None, None, None, None]]`

Does anyone know why?

**Comment by jsvine** [comment]
Hi @chengtie, and very interesting parsing strategy. Have you tried examining the problem PDF with the [visual debugging tools](https://github.com/jsvine/pdfplumber#visual-debugging)?

E..g,:

```python
im = page.to_image()
im.draw_lines(page.vertical_edges)
```

... and

```python
im.reset().debug_tablefinder()
```

Those images should show automatically in a notebook, but you can also call `im.show()` in a script or REPL.
